javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,46 @@
+// Housey pricey (A house price estimator)
+// Peter
+
+const housePeter = {
+  name: "Peter",
+  width: 8,
+  depth: 10,
+  heigth: 10,
+  gardenSizeInM2: 100,
+  houseCosts: 2500000,
+
+  housePriceCalculation: function () {
```

Comment:
I like the idea of putting function into the object, overall that's well known object-oriented-programming approach. 
But there's one thing to improve here: to avoid copying the function two times it makes sense to declare it at the top of the file:
e.g. 
function housePriceCalculation() {
    let volumeInMeters = .....
   ....
}

housePeter = {
     ....
     houseCosts: 2500000,
     housePriceCalculation: housePriceCalculate
}
houseJulia = {
     ....
     houseCosts: 1000000,
     housePriceCalculation: housePriceCalculate
}